contest/static/contest/js/contest.js

```diff
@@ -24,22 +24,6 @@ $('#myTab a').click(function(e) {
     $('#myTab a[href="#scoreboard"]').tab('show')
     $('#myTab a[href="#status"]').tab('show')
 })
-$('#right-panel-link').panelslider({
-    side: 'right'
-});
-
-function startTime() {
-    var today = new Date();
-    var h = today.getHours();
-    var m = today.getMinutes();
-    var s = today.getSeconds();
-    m = checkTime(m);
-    s = checkTime(s);
-    document.getElementById('clock').innerHTML = h + ":" + m + ":" + s;
-    var t = setTimeout(function() {
-        startTime()
-    }, 500);
-}
 
 function checkTime(i) {
```

Comment:
I think django may have some tag for time format string, if so, you don't have to do that yourself.